src/utility/VersionUtility.ts

```diff
@@ -48,25 +48,17 @@ export class VersionUtility {
     }
 
     // Returns the CLI version that this VSCode session expects
-    public static async getExpectedCliVersionAsync(context: vscode.ExtensionContext, experimentationService: IExperimentationService, packageJsonContent: object, logger: Logger): Promise<string> {
-        const bridgeEnvironment = EnvironmentUtility.getBridgeEnvironment(logger);
-
-        // This call will return the upcoming version for treatment group; control & everyone else will get the stable prod version. Undefined if the flight is currently inactive.
-        // TODO(ansoedal): Disable ExP permanently. For now this call will return undefined, as long as the treatment variable doesn't exist
-        const expBinariesVersion = experimentationService.getTreatmentVariable<string>(/*configId*/ `vscode`, /*name*/ `mindaroBinariesVersion-doesnotexist`);
-        logger.trace(`Call to ExP returned version '${expBinariesVersion}'`);
+    public static async getExpectedCliVersionAsync(packageJsonContent: object): Promise<string> {
+        const bridgeEnvironment = EnvironmentUtility.getBridgeEnvironment();
 
         // How we resolve the expected CLI version:
         //  1. Is the BRIDGE_CLI_VERSION environment variable set? Use that one.
         //  2. Is a flight in progress? Get the binaries version from ExP.
         //  3. Fall back onto whichever LKS version was specified in package.json. (For dev and staging, this is whichever version matches this extension. For Prod, this is a variable we specify in the Mindaro-Connect-VSCode pipeline.)
         const packageJsonVersion: string = packageJsonContent[`extensionMetadata`][`expectedCLIVersion`];
         const expectedCLIVersion = process.env.BRIDGE_CLI_VERSION != null ? process.env.BRIDGE_CLI_VERSION :
-            (expBinariesVersion != null && bridgeEnvironment === Environment.Production ? expBinariesVersion :
-            (process.env.BRIDGE_BUILD_PATH != null ? null :
-            (packageJsonVersion != null && !packageJsonVersion.startsWith(`$`) ? packageJsonVersion : null)));
+            (packageJsonVersion != null && !packageJsonVersion.startsWith(`$`) ? packageJsonVersion : null);
```

Comment:
@Tatsinnit you will have conflicts with Vidya's branch for this method, she has refactored this method to make it simpler.

Reply:
☕️ 💡 nice! yeah, did see how in other PR, I was/am keen to get the **unit-test** changes into PR mode just to start the ball rolling for clearing unwanted parameters and removing other unnecessary parameters et al. (Especially discussion around this change as well)

Reply:
@Tatsinnit we can ask Vidya to make similar changes in her PR to avoid this conflict. are you okay doing this particular changes in her PR instead ? @Vidya2606 FYI :)